react/src/Navigation.tsx

```diff
@@ -264,14 +268,18 @@ export default function Navigation({
                     </div>
                     <Divider />
                     <List>
-                        {routes.map((route, index) => (
-                            <ListItemRouterLink
-                                key={index}
-                                to={route.linkTo ? route.linkTo : "" + route.path}
-                                primary={route.pageName}
-                                children={route.icon}
-                            />
-                        ))}
+                        {routes.map((route, index) =>
+                            !route.requiresAdmin || isAdmin ? (
+                                <ListItemRouterLink
+                                    key={index}
+                                    to={route.linkTo ? route.linkTo : "" + route.path}
+                                    primary={route.pageName}
+                                    children={route.icon}
+                                />
+                            ) : (
+                                <></>
```

Comment:
```suggestion
                                <React.Fragment key={index} />
```
This seems silly but it'll complain the console otherwise since it can't figure out what component the empty item originally was.

Reply:
🤔  That is silly